### PR TITLE
[AC-441] Convert Hardcoded Strings to String Resources

### PR DESCRIPTION
**Pull request by ribhavsharma**
## Description of what I changed
Converted all hardcoded strings to String Resources, so that the app can have support for multiple languages in the future.

## Task that I worked on
GCI Task: [Android Client: Convert Hardcoded Strings to String Resources](https://codein.withgoogle.com/dashboard/task-instances/6652232368390144/)
## Issue that I worked on
[AC-441](https://issues.openmrs.org/browse/AC-441)
## Checklist: I completed these to help reviewers :)
- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -390,5 +390,10 @@
     <string name="patients_tab_title">Patients</string>
     <string name="failed_provider_details">Failed to get Provider details!</string>
     <string name="retry_action">Retry</string>
+    <string name="surgeon">Surgeon</string>
+    <string name="provider_provider">Provider - Provider Relationship</string>
+    <string name="provider_patient">Provider - Patient Relationship</string>
+    <string name="jake_smith">Jake Smith</string>
+    <string name="doctor">Doctor</string>
```
Comment (PrathameshMutkure) [inline openmrs-client/src/main/res/values/strings.xml]: Hii, can you please add comments above the strings mentioning to which layout they belong to like done for other strings in the file.

Wait for any mentor to have a say!

Comment (ribhavsharma) [reply openmrs-client/src/main/res/values/strings.xml]: fixed it. Thanks for pointing it out 😄

**Comment by ribhavsharma** [comment]
> @ribhavsharma Please follow the [Pull Request Guidelines](https://wiki.openmrs.org/display/docs/Pull+Request+Tips).
> You need to include the JIRA ticket number in the Pull Request Name, and also have a link to it in the description.
> Also, please squash your commits into a single commit.
@vansha10 Kindly check out the PR now.

**Comment by vansha10** [comment]
Good work @ribhavsharma.
Just one more thing, you need to include the JIRA ticket number in the Pull Request Title. Please replace 
_[GCI '19]_ with _AC 441_.

**Comment by vansha10** [comment]
Looks good @ribhavsharma.
@f4ww4z Can you have a look at this?

**Comment by f4ww4z** [comment]
Looks @ribhavsharma ! Sorry for the delay.

**Pull request merged**